dashboard/src/components/status/shard-store.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2016-2020 the original author or authors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Shard } from "@/components/status/shard";
+import fetcher from "@/fetcher";
+
+export const FETCH_SHARDS = "FETCH_SHARDS";
+
+const FETCH_SHARDS_INTERNAL = "FETCH_SHARDS_INTERNAL";
+
+const LOAD_SHARDS = "LOAD_SHARDS";
+const FETCHING_SHARDS = "FETCHING_SHARDS";
+
+export const shardStore = {
+	namespaced: true,
+	modules: {},
+	state: () => ({
+		shardsLoading: false, //true each time there is a request for staff in flight
```

Comment:
solid copy paste development but you missed the "staff" comment 🙃

Reply:
Oops.